libc/docs/dev/header_generation.rst

```diff
@@ -23,6 +23,7 @@ Instructions
 
 Required Versions:
   - Python Version: 3.11.8
+  - PyYaml Version: 5.1
```

Comment:
Actually, change this to PyYAML

Reply:
Any luck bringing down the Python version? Python 3.8 worked previously, so increasing that is a lot of the problems.

Reply:
we support 3.6 as min version now if that works.

Reply:
@petrhosek let me know if this is good for Fuchsia